src/main/java/Event.java

```diff
@@ -0,0 +1,20 @@
+public class Event extends Todo {
+    private final String TYPE_SYMBOL = "[E]";
+    private final String FROM;
+    private final String TO;
+    public Event(String task, String from, String to) {
```

Comment:
Would it be clearer to have a linespace between TO variable and Event constructor to enhance clarity?